ci/do_ci.ps1

```diff
@@ -14,41 +14,53 @@ function bazel_binary_build($type) {
 }
```

Comment:
For the maintainers to consider: should we change over our CI pipelines to use the `do_ci` pattern like on Linux? (or should we just simply keep it around until we can do the full set of builds/tests?)

Reply:
Perhaps we defer this discussion until this is merged? Good conversation, and probably doesn't have to hold up adoption of this specific PR.